README.md

```diff
@@ -8,196 +8,29 @@ about the [Universal AST](https://doc.bblf.sh/) of the code itself. gitbase is b
 gitbase implements the *MySQL* wire protocol, it can be accessed using any MySQL
 client or library from any language.
 
+[src-d/go-mysql-server](https://github.com/src-d/go-mysql-server) is the project where the SQL engine used by ***gitbase*** is implemented.
```

Comment:
I wouldn't use bold and italics at the same time. Maybe I'm a bit picky but gitbase in this context doesn't seem worth emphasizing.

Reply:
s/is the project where the SQL engine used by gitbase is implemented/is the SQL engine implementation used by gitbase